video_xblock/backends/brightcove.py

```diff
@@ -470,7 +463,7 @@ def get_default_transcripts(self, **kwargs):
                 ]
             message (str): message for a user on default transcripts fetching.
         """
-        if not self.api_key and self.api_secret:
+        if not self.api_key and not self.api_secret:
```

Comment:
Yeah, second `not` or rather parentheses. I.e. `not (self.api_key and self.api_secret)`